### DISCUSSION TITLE
Missing Code of conduct

### DISCUSSION THREAD

**Discussion by BobinMathew** [btcpayserver/btcpayserver#2757] [General]
This is missing a Code of conduct file.
Can I give PR?

**Comment by pavlenex** [comment]
We discussed this a while ago and decided it's not really needed. Why do you think it's important? By doing a PR would you just be providing a standard template? 

If we were ever to do this which I don't really think we need, it should be really adjusted and well-written to specifically explain the community, and that kind of document can only been written by someone who's been here for a while. 

So concept NACK on my end, but happy to hear your reasoning on why this matters.

**Comment by bolatovumar** [comment]
I propose this Code of Conduct:

1. Obey The Emperor at all times
2. Heed the advice of Uncle Rockstar
3. Buy Argoneum!
